### ISSUE TITLE
Please mark annotations as @Inherited

### ISSUE THREAD

**Issue by seanf** [BinaryTweed/quarantining-test-runner#4]
If the `@Quarantine` and `@DelegateRunningTo` annotations are marked as `@Inherited`, test classes will have the option of extending another class which provides the annotations in a common place.

eg

```
@Quarantine({"org.jboss.seam"})
@RunWith(QuarantiningRunner.class)
public abstract class AbstractSeamTest {}

public class MySeamTest extends AbstractSeamTest {...}
```

**Comment by BinaryTweedDeej** [comment]
Fixed in 0.0.3

**Comment by seanf** [comment]
Thanks @BinaryTweedDeej !
